spec/factories/course_assessment_answers.rb

```diff
@@ -3,9 +3,26 @@
     transient do
       course { build(:course) }
       assessment { build(:assessment, course: course) }
+      submission_traits []
     end
 
-    submission { build(:course_assessment_submission, assessment: assessment) }
+    submission { build(:course_assessment_submission, *submission_traits,
+                         assessment: question.assessment) }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 10, 59 should be on its own line.